rnote-ui/src/workspacebrowser/filerow/actions/add_dir.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::PathBuf;
+
+use crate::workspacebrowser::{filerow::widget_helper, FileRow};
+use gettextrs::gettext;
+use gtk4::{
+    gio, glib,
+    glib::clone,
+    pango,
+    prelude::FileExt,
+    traits::{BoxExt, ButtonExt, EditableExt, PopoverExt, StyleContextExt, WidgetExt},
+    Align, Button, Entry, Label, Popover,
+};
+
+impl FileRow {
+    pub fn add_dir_action(&self) -> gio::SimpleAction {
```

Comment:
these methods should be prefixed with `create_` to indicate that they are not existing properties of the widget, but constructors.

Reply:
done